CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 This file is a history of the changes made to @idearium/cli.
 
+## Unreleased
+
+- New commands.
+
+## Improvements.
+
+- Added `c mk delete`, `c mk hosts`, `c mk ip`, `c mk start`, `c mk stop` and `c mk restart` to control Minikube.
+- Added `c hosts add`, `c hosts get` and `c hosts remove` to help with easy hosts management.
+- Added `c project env ls`, `c project env get` and `c project env set`, `c project init`, `c project name`, `c project organisation`, `c project prefix` to help with project environment management.
+- Added a `c d build` command.
+- Added `c kc apply`, `c kc build`, `c kc clean`, `c kc cmd`, `c kc context`, `c kc context get`, `c kc context set`, `c kc deploy`, `c kc logs`, `c kc pod`, `c kc start` and `c kc stop` commands to help with kubectrl.
```

Comment:
I think it should be `kubectl`? Maybe do a search and replace, it is in a few spots.